packages/paste-core/utilities/flex/__tests__/flex.test.tsx

```diff
@@ -207,6 +207,17 @@ describe('Flex Display', () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('it should render as any HTML element', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Flex as="article">child</Flex>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
Would prefer an enzyme test here to check the tag https://airbnb.io/enzyme/docs/api/ShallowWrapper/name.html#examples

Reply:
Good call, went with both